api/libproject/devstart.rb

```diff
@@ -286,13 +289,15 @@ def drop_cloud_cdr(*args)
 end
 
 def connect_to_cloud_db(*args)
-  GcloudContext.new("connect-to-cloud-db", args, true).run do |ctx|
-    pw = ENV["WORKBENCH_DB_PASSWORD"]
-    # TODO Switch this to run_inline once Common supports redaction.
-    run_with_redirects(
-        "mysql -u \"workbench\" -p\"#{pw}\" --host 127.0.0.1 "\
-        "--port 3307 --database #{ENV["DB_NAME"]}",
-        to_redact=pw)
+  op = WbOptionsParser.new("connect-to-cloud-db", args)
+  CloudSqlProxyContext.new(GcloudContextV2.new(op)).run do |env_file_path|
+    sleep 1 # TODO(dmohs): Detect running better.
+    env = Hash[File.read(env_file_path).split(/\n/).map {|l| l.split(/=/)}]
```

Comment:
We had a read_db_vars function that did this before, used in multiple places. 

If you want to move away from using it, please add a replacement.